spec/API_specification/creation_functions.md

```diff
@@ -57,7 +57,7 @@ Convert the input to an array.
 
 #### Parameters
 
--   **obj**: _Union\[ float, NestedSequence\[ bool | int | float ], SupportsDLPack, SupportsBufferProtocol ]_
+-   **obj**: _Union\[ &lt;array&gt;, float, NestedSequence\[ bool | int | float ], SupportsDLPack, SupportsBufferProtocol ]_
```

Comment:
I can raise an issue/PR if more appropriate, but while you're at it—as the docs say `obj` can be a Python scalar,  am I correct in saying that `bool` and `int` should be part of the top-level `Union` type hint alongside `float`?

Reply:
Technically `bool` and `int` are already covered by `float` (they sub-types), so the main rule we have used for determining whether or not to add `bool` and/or `int` explicitly is: does it help for clarity?

In this case the output dtype is different for `bool` or `int` scalar input, so makes sense to add explicitly.